README.md

```diff
@@ -96,21 +114,29 @@ Invoke will handle establishing local virtual environments, etc. Task definition
     ```bash
     gcloud builds submit \
         --config test/advance.cloudbuild.yaml \
-        --substitutions 'SHORT_SHA=manual'
+        --substitutions 'COMMIT_SHA=manual,REPO_NAME=manual'
```

Comment:
I tested the README steps and it all worked after I made this change.

Reply:
Thank you!